File: src/main/java/com/github/afzalive/firstintellijplugin/ui/AppActionsToolWindowFactory.java
package com.github.afzalive.firstintellijplugin.ui;

import com.intellij.openapi.project.Project;
import com.intellij.openapi.wm.ToolWindow;
import com.intellij.openapi.wm.ToolWindowFactory;
import com.intellij.ui.content.Content;
import com.intellij.ui.content.ContentFactory;
import org.jetbrains.annotations.NotNull;

public final class AppActionsToolWindowFactory implements ToolWindowFactory {
    public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindow toolWindow) {
        Content content = ContentFactory.getInstance().createContent(new AppActionsPanel(project), "", false);
        toolWindow.getContentManager().addContent(content);
    }
}

File: src/main/java/com/google/assistant/plugin/appactions/ui/toolwindow/AppActionsToolWindowFactory.kt
package com.google.assistant.plugin.appactions.ui.toolwindow

import com.intellij.openapi.project.Project
import com.intellij.openapi.wm.ToolWindow
import com.intellij.openapi.wm.ToolWindowFactory
import com.intellij.ui.content.ContentFactory

/* compiled from: AppActionsToolWindowFactory.kt */ /* loaded from: google-assistant-plugin-2.5.1.zip:google-assistant-plugin/lib/app-actions-test-tool-kotlin-2.5.1.jar:com/google/assistant/plugin/appactions/ui/toolwindow/AppActionsToolWindowFactory.class */
class AppActionsToolWindowFactory() : ToolWindowFactory {
    private var appActionsToolWindowPanel: AppActionsPanel? = null

    constructor(appActionsToolWindowPanel: AppActionsPanel) : this() {
        this.appActionsToolWindowPanel = appActionsToolWindowPanel
    }

    override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
        val contentFactory = ContentFactory.getInstance()
        val content = contentFactory.createContent(appActionsToolWindowPanel, "", false)
        toolWindow.contentManager.removeAllContents(true)
        toolWindow.contentManager.addContent(content)
    }
}
